package.json

```diff
@@ -66,9 +67,9 @@
     "json-loader": "0.5.4",
     "nightwatch-autorun": "3.1.0",
     "parallelshell": "2.0.0",
+    "react": "15.5.4",
```

Comment:
Also peer dependency should be changed to - `"react": ">=15.3"`, because of the `PureComponent`